start.go

```diff
@@ -202,8 +202,7 @@ command(s) that get executed on start, edit the args parameter of the spec. See
 				"--state-dir", root,
 				"--listen", filepath.Join(namespace, "namespaced.sock"),
 			)
-			cmd = exec.Command("runv", args...)
-			cmd.Path = path
+			cmd = exec.Command(path, args...)
```

Comment:
@Crazykev  Yeah, as I metioned before, now in ocid they don't configure cmd.Env, we inherit all environments from ocid include PATH.
There will not be any problem for us to find qemu or any thing else in PATH, so everything is fine:)